sunpy/database/tests/test_tables.py

```diff
@@ -509,8 +509,8 @@ def test_create_display_table():
         'observation_time_start', 'instrument', 'size',
         'wavemin', 'path', 'starred', 'tags']
     table = _create_display_table(entries, columns)
-    filedir = os.path.dirname(os.path.realpath(__file__))
-    with open(os.path.join(filedir, 'test_table.txt'), 'r') as f:
+    filedir = Path(Path(__file__).resolve()).parent
```

Comment:
Do we not have one too many `Path(` here?

Reply:
sorry but what do you mean by that

Reply:
You call `Path` too many times from the looks of it.

Reply:
umm sorry, but i guess in this instance, itll work like that only. I tried minimizing as much as possible.

Reply:
I will have to check but it feels wrong at least. 

Would `Path(__file__).resolve().parent` or `Path(__file__).parent` not work?